packages/Algernon.package/ALGDynamicWorldMenuType.class/class/filterOutDuplicates..st

```diff
@@ -0,0 +1,11 @@
+algertem lists
+filterOutDuplicates: anOrderedCollection
+
+	^ anOrderedCollection reject: [ :entry | 
+		entry isMenuItemMorph
+			ifTrue: [
+				ALGStaticWorldMenuType algertemList anySatisfy: 
+				[ :entry2 |
+					entry2 name = entry contents                                      ]
+																			]
```

Comment:
```suggestion
			ifTrue: [
				ALGStaticWorldMenuType algertemList anySatisfy: 
					[ :staticEntry |
						staticEntry name = entry contents ]
																			]
			ifFalse: [ false ]
```

I find this more intuitive. Also, I think the linter would be mad at us with the original code - But, well, ... As we already found out earlier, its linting skills on the class side are just too bad